lib/Support/ErrorHandling.cpp

```diff
@@ -85,7 +92,15 @@ static void klee_vfmessage(FILE *fp, const char *pfx, const char *msg,
                        /*bg=*/false);
   }
 
-  fdos << "KLEE: ";
+  if (LogCpuTime) {
+    sys::TimeValue now(0,0), user(0,0), sys(0,0);
+    sys::Process::GetTimeUsage(now,user,sys);
+    fdos << (user.seconds() + sys.seconds()) << ".";
+    fdos << (user.nanoseconds() + sys.nanoseconds());
```

Comment:
This won't include the time from forked processes though. KLEE forks a process by default when using STP. When using Z3 there is no forked process.